pkg/models/change_table_record.go

```diff
@@ -38,3 +96,14 @@ type FieldValue struct {
 	New interface{} `json:"new"`
 	Old interface{} `json:"old"`
 }
+
+// NameCamelCase converts the name database string value, to lower case camel. This is meant as an approximation of the GQL field name
+func (field *Field) NameCamelCase() string { //can include ctx context.Context if desired
```

Comment:
Does it make sense to include ctx here? If there is no present need we should remove the comment.

Reply:
Thanks @OddTomBrooks, I don't think we need it here, but I wanted to note that it is an allowed signature if we need it. There is no need to log anything in this method or use anything from context, but I added the note since this largely started as a discovery task.